src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_TryCatch.cs

```diff
@@ -61,10 +61,8 @@ void M(int i)
               ILiteralOperation (OperationKind.Literal, Type: System.Int32, Constant: 0) (Syntax: '0')
         Handler: 
           IBlockOperation (1 statements) (OperationKind.Block, Type: null) (Syntax: '{ ... }')
-            IExpressionStatementOperation (OperationKind.ExpressionStatement, Type: null) (Syntax: 'throw ex;')
-              Expression: 
-                IThrowOperation (OperationKind.Throw, Type: System.Exception) (Syntax: 'throw ex;')
-                  ILocalReferenceOperation: ex (OperationKind.LocalReference, Type: System.Exception) (Syntax: 'ex')
+            IThrowOperation (OperationKind.Throw, Type: null) (Syntax: 'throw ex;')
```

Comment:
With this change, the type of the Exception being thrown is no longer in the Operation tree. This seems like a mistake, we should at least set the Type here to be the type of the underlying exception.

Reply:
> With this change, the type of the Exception being thrown is no longer in the Operation tree. This seems like a mistake, we should at least set the Type here to be the type of the underlying exception.

Neither thow expression, nor throw statement should have a type, the expression being thrown has it.